core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -163,15 +163,14 @@ public Options fetch(ColumnType... colsToFetch) {
           case FLUSH_ID:
             qualifiers.add(FLUSH_COLUMN);
             break;
-          case LAST:
-            families.add(LastLocationColumnFamily.NAME);
-            break;
           case LOADED:
             families.add(BulkFileColumnFamily.NAME);
             break;
           case LOCATION:
             families.add(CurrentLocationColumnFamily.NAME);
             families.add(FutureLocationColumnFamily.NAME);
+            families.add(LastLocationColumnFamily.NAME);
+            families.add(TabletsSection.SuspendLocationColumn.SUSPEND_COLUMN.getColumnFamily());
```

Comment:
```suggestion
            families.add(SuspendLocationColumn.SUSPEND_COLUMN.getColumnFamily());
```

Reply:
I applied this suggestion manually to the case for SUSPEND.